SAKPaaS/src/app/components/location-panel/location-panel.component.scss

```diff
@@ -2,19 +2,18 @@
   z-index: 3;
   display: flex;
   flex-direction: column;
-  padding: 14px 14px 0 14px;
+  padding: 7px 10px 10px 6px;
```

Comment:
messed up the order.
you need: `padding: 10px 6px 0 7px`

Reply:
Meinst du 10 6 10 7 oder warum die 0?

Reply:
Because on the bottom there is no need for extra padding.